qcfractal/queue/managers.py

```diff
@@ -548,49 +548,56 @@ def update(self, new_tasks: bool = True, allow_shutdown=True) -> bool:
 
         open_slots = max(0, self.max_tasks - self.active)
 
-        if (new_tasks is False) or (open_slots == 0):
-            return True
-
-        # Get new tasks
-        payload = self._payload_template()
-        payload["data"]["limit"] = open_slots
-
         # Crunch Statistics
         self.statistics.total_failed_tasks += n_fail
         self.statistics.total_successful_tasks += n_success
         self.statistics.total_core_hours += task_cpu_hours
         na_format = ''
         float_format = ',.2f'
         if self.statistics.total_completed_tasks == 0:
-            success_rate = "(N/A yet)"
-            success_format = na_format
+            task_stats_str = "Task statistics unavailable until first tasks return"
```

Comment:
Can this simply be `N/A` or `N/A, no tasks complete`?

Reply:
I changed the way the logic works here, so if that block executes, only the "Task statistics unavailable..." line shows and the rest never trigger. This was based on #336 where a more helpful initial logger message was requested.